NEFI_functions/check_neon_products.r

```diff
@@ -0,0 +1,55 @@
+#This function takes a data frame that has site and year_month columns and a vector of NEON data products of interest.
+#It returns a dataframe of unique site X year_month combinations from your data and whether or not there are data for it.
+
+#testing function.
+#clear R environment, load packages.
+#rm(list=ls())
+#library(nneo)
+#library(data.table)
+#map <- readRDS('/fs/data3/caverill/NEFI_microbial/map_otu/16S_map_clean.rds')
+#of_interest <- c('DP1.10078.001','DP1.10086.001')
+#test <- check_neon_products(map,of_interest)
+#head(test)
```

Comment:
I'm going to suggest that function documentation use formal ROxygen tags. The easiest way to get started on this in RStudio is to move the cursor to the function and use Code > Insert ROxygen Skeleton which will stick in the basic tags above the function and then you can add text (description, variable definitions,  example, etc) and any other tags. The second half of the 'testing' bit you have above would make a good example. 

A quick intro to ROxygen is here http://kbroman.org/pkg_primer/pages/docs.html and a somewhat longer example is here http://r-pkgs.had.co.nz/man.html and the full list of tags is in the roxygen2 package documentation http://roxygen.org/roxygen2-manual.pdf

Reply:
done!